go/oauth2util/util.go

```diff
@@ -0,0 +1,75 @@
+package oauth2util
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+)
+
+// Default 3LO authorization handler. Prints the authorization URL on stdout
```

Comment:
Formatting of godoc stars with the symbol name.

// defaultAuthorizeFlowHandler prints the authorization URL on stdout...